workshops/test/test_person.py

```diff
@@ -947,9 +951,12 @@ def test_all_requirements_satisfied(self):
                          set())
 
     def test_some_requirements_are_fulfilled(self):
+        # Homework was accepted, the second time.
```

Comment:
Is there still a test somewhere for "a single passed homework object was created"? Also, how are you testing that everything worked - just by the fact that there's no exception thrown?